pkgs/os-specific/linux/firmware/fwupd/install-fwupdplugin-to-out.patch

```diff
@@ -1,37 +1,48 @@
 diff --git a/libfwupdplugin/meson.build b/libfwupdplugin/meson.build
-index 1afa28e1..3da81d30 100644
+index 71667b6bc..b944d97f2 100644
 --- a/libfwupdplugin/meson.build
 +++ b/libfwupdplugin/meson.build
-@@ -220,7 +220,8 @@ fwupdplugin = library(
+@@ -250,7 +250,7 @@ fwupdplugin = library(
    ],
    link_args: cc.get_supported_link_arguments([vflag]),
    link_depends: fwupdplugin_mapfile,
--  install: true
-+  install: true,
-+  install_dir: bindir / '..' / 'lib',
+-  install_dir: libdir_pkg,
++  install_dir: bindir / '..' / 'lib' / f'fwupd-@fwupd_version@',
+   install: true
  )
  
- fwupdplugin_pkgg = import('pkgconfig')
-@@ -280,7 +281,8 @@ if introspection.allowed()
-       girtargets,
-       fwupd_gir[0],
-     ],
--    install: true
-+    install: true,
-+    install_dir_typelib: bindir / '..' / 'lib' / 'girepository-1.0',
-   )
- 
-   # Verify the map file is correct -- note we can't actually use the generated
-diff --git a/meson.build b/meson.build
-index b91dd037..f97b4c26 100644
---- a/meson.build
-+++ b/meson.build
-@@ -504,7 +504,7 @@ if build_standalone
- if host_machine.system() == 'windows'
-   plugin_dir = 'fwupd-plugins-@0@'.format(libfwupdplugin_lt_current)
- else
--  plugin_dir = join_paths(libdir, 'fwupd-plugins-@0@'.format(libfwupdplugin_lt_current))
-+  plugin_dir = join_paths(bindir, '..', 'lib', 'fwupd-plugins-@0@'.format(libfwupdplugin_lt_current))
- endif
- conf.set_quoted('FWUPD_PLUGINDIR', plugin_dir)
- endif
+diff --git a/plugins/modem-manager/meson.build b/plugins/modem-manager/meson.build
+index 325c55e05..290dbdb72 100644
+--- a/plugins/modem-manager/meson.build
++++ b/plugins/modem-manager/meson.build
+@@ -26,7 +26,7 @@ shared_module('fu_plugin_modem_manager',
+   include_directories: plugin_incdirs,
+   install: true,
+   install_rpath: libdir_pkg,
+-  install_dir: libdir_pkg,
++  install_dir: bindir / '..' / 'lib' / f'fwupd-@fwupd_version@',
+   c_args: cargs,
+   link_with: plugin_libs,
+   dependencies: [
+diff --git a/src/meson.build b/src/meson.build
+index ebb262224..19e333972 100644
+--- a/src/meson.build
++++ b/src/meson.build
+@@ -87,7 +87,7 @@ fwupdutil = library(
+   ],
+   install: true,
+   install_rpath: libdir_pkg,
```

Comment:
I wonder if also replacing this would allow us to drop `-Dc_link_args`.

Reply:
Good point